openex-api/src/main/java/io/openex/rest/user/form/login/LoginUserInput.java

```diff
@@ -1,30 +1,24 @@
 package io.openex.rest.user.form.login;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import jakarta.validation.constraints.NotBlank;
+import lombok.*;
 
 import static io.openex.config.AppConfig.MANDATORY_MESSAGE;
 
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class LoginUserInput {
 
     @NotBlank(message = MANDATORY_MESSAGE)
+    @JsonProperty("user_login")
```

Comment:
Why did you change the property names to json?
I think that break the login page via the front end.

Reply:
Thanks!! I fix it!